build.gradle.kts

```diff
@@ -28,3 +28,9 @@ tasks.test {
 kotlin {
     jvmToolchain(8)
 }
+
+ktlint {
```

Comment:
why are we excluding this class from ktlint?

Reply:
Ideally, Enum entry name should be uppercase underscore-separated names or upper camel-case but the possible values of status do not meet that criteria and so the class was being flagged

Reply:
we can annotate it with @SerializedName 
see from here. https://stackoverflow.com/questions/45553339/how-use-kotlin-enum-with-retrofit

Another thoughts i have, can we separate the prediction class from the API layer? so the user only interacts with the   domain object and not dto